<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertGroup</class>
 <widget class="QDialog" name="InsertGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QLabel" name="Infolabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>请点击查看班级已有小组</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>班级</string>
   </property>
  </widget>
  <widget class="QComboBox" name="classId">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="groupId">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>组号</string>
   </property>
  </widget>
  <widget class="QToolButton" name="ok">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QToolButton" name="out">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="Warnlabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>请选择班级并输入要添加的小组组号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>当前小组</string>
   </property>
  </widget>
  <widget class="QComboBox" name="groupCur">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>out</sender>
   <signal>clicked()</signal>
   <receiver>InsertGroup</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
